tests/test_validators.py

```diff
@@ -48,3 +52,257 @@ def test_value_valid(self):
     def test_value_invalid_type(self):
         with self.assertRaises(exceptions.FieldError):
             self.validate(fake.word())
+
+
+class ValidatePositiveNumberValueTestCase(unittest.TestCase):
```

Comment:
I don't mean to be picky, but I would prefer using `self.assertTrue` / `self.assertFalse`  in the tests - I perceive it more readable.

Reply:
Done